src/vs/workbench/parts/preferences/browser/settingsTree.ts

```diff
@@ -1338,7 +1338,7 @@ export class SettingsTree extends NonExpandableTree {
 			if (foregroundColor) {
 				// Links appear inside other elements in markdown. CSS opacity acts like a mask. So we have to dynamically compute the description color to avoid
 				// applying an opacity to the link color.
-				const fgWithOpacity = new Color(new RGBA(foregroundColor.rgba.r, foregroundColor.rgba.g, foregroundColor.rgba.b, .7));
+				const fgWithOpacity = new Color(new RGBA(foregroundColor.rgba.r, foregroundColor.rgba.g, foregroundColor.rgba.b, .9));
```

Comment:
@roblourens FYI, as we discussed, I updated the opacity for the secondary text and removed the description opacity from the css (since it was being applied twice, causing a cc failure)